src/components/FindWalletProductTable/WalletSubComponent.tsx

```diff
@@ -0,0 +1,188 @@
+import { useRouter } from "next/router"
+import { useTranslation } from "react-i18next"
+import { FaDiscord, FaGlobe, FaXTwitter } from "react-icons/fa6"
+import { MdInfoOutline } from "react-icons/md"
+
+import { FilterOption, Lang, WalletData } from "@/lib/types"
+
+import { WalletFilters } from "@/components/FindWalletProductTable/data/WalletFilters"
+import {
+  GreenCheckProductGlyphIcon,
+  WarningProductGlyphIcon,
+} from "@/components/icons/staking"
+import InlineLink from "@/components/Link"
+import Tooltip from "@/components/Tooltip"
+
+import { cn } from "@/lib/utils/cn"
+import { trackCustomEvent } from "@/lib/utils/matomo"
+import { getLocaleFormattedDate } from "@/lib/utils/time"
+
+const SocialLink = (props) => (
+  <InlineLink
+    display="flex"
+    height={6}
+    alignItems="center"
+    verticalAlign="middle"
+    transform="scale(1)"
+    transition="transform 0.1s"
+    _hover={{
+      transform: "scale(1.15)",
+    }}
+    {...props}
+  />
+)
+
+interface WalletSubComponentProps {
+  wallet: WalletData
+  filters: FilterOption[]
+  listIdx: number
+}
+
+const WalletSubComponent = ({
+  wallet,
+  filters,
+  listIdx,
+}: WalletSubComponentProps) => {
+  const { locale } = useRouter()
+  const { t } = useTranslation("page-wallets-find-wallet")
+  const walletFiltersOptions: FilterOption[] = WalletFilters()
+
+  const walletFilterDisplayOrder = [
+    t("page-find-wallet-features"),
+    t("page-find-wallet-security"),
+    `${t("page-find-wallet-buy-crypto")} / ${t("page-find-wallet-sell-for-fiat")}`,
+    t("page-find-wallet-smart-contract"),
+    t("page-find-wallet-advanced"),
+  ]
+
+  const walletLastUpdated = getLocaleFormattedDate(
+    locale as Lang,
+    wallet.last_updated
+  )
+
+  trackCustomEvent({
+    eventCategory: "WalletMoreInfo",
+    eventAction: "More info wallet",
+    eventName: `More info ${wallet.name}`,
+  })
+
+  return (
+    <div className="flex flex-row gap-2">
+      <div className="w-1 md:w-14">
+        <div
+          className={cn(
+            `m-auto h-full w-1 bg-gradient-to-b ${wallet.brand_color} to-[rgba(217, 217, 217, 0)] to-97%`
+          )}
```

Comment:
`[rgba(217, 217, 217, 0)]` = `transparent` = Default for both `from-` and `to-` colors once other tailwind gradient classes have been applied. (`from-0% from-transparent via-50% via-transparent to-100% to-transparent` are the defaults to my knowledge)

And instead of string interpolating here, we should add `wallet.brand_color` as a second argument to `cn()` (similar to `className`)

```suggestion
          className={cn(
            "m-auto h-full w-1 bg-gradient-to-b to-97%",
            wallet.brand_color
          )}
```